### DISCUSSION TITLE
cloudinary

### DISCUSSION THREAD

**Discussion by tinect** [FriendsOfShopware/FroshPlatformThumbnailProcessor#69] [Patterns]
This pattern is for [cloudinary](https://cloudinary.com/)
My opinion to this service: has free plan, fast and including webp and avif

```
https://res.cloudinary.com/MYID/image/fetch/w_{width},f_auto/https://{mediaUrl}/{mediaPath}
```

**Comment by Oli-T** [comment]
Looks like the URL is missing. This worked for me:
`https://res.cloudinary.com/MYID/image/fetch/w_{width},f_auto/https://{mediaUrl}/{mediaPath}`

**Comment by markusgabriel** [comment]
Ok, for me to make it work I had to use the URL like this:

`https://res.cloudinary.com/<product_environment_name>/image/fetch/w_{width},f_auto/{mediaUrl}/{mediaPath}`

My mediaUrl variable already had the https:// in it, so I removed it.

So I created a free Cloudinary account, copied the product environment name and restricted the access to my domain. Then I installed the FroshPlatformThumbnailProcessor plugin and pasted the url in the config section. 

I had trouble with serving webP images, so I changed `f_auto` to `f_webp` in the url above.

Now everything works! 🥳
